### ISSUE TITLE
[a11y] : Search component does not register character input when macOS VoiceOver is active

### ISSUE THREAD

**Issue by jorienoll** [carbon-design-system/carbon#19349]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.78.2

### React version

17.0.2

### Automated testing tool and ruleset

IBM Accessibility Requirements

### Assistive technology

macOS VoiceOver

### Description

When macOS VoiceOver is turned on, the <Search /> component fails to register character key input unless the user presses Space or Backspace. No other keys trigger the expected onKeyDown or input behavior, effectively blocking screen reader users from typing into the field using a keyboard.

### WCAG 2.1 Violation

2.1.1 Keyboard

### Reproduction/example

@carbon/react template is broken: [Error: Build failed with 1 error: node_modules/@carbon/react/es/components/PageHeader/PageHeader.js:22:9: ERROR: No matching export in "node_modules/@carbon/utilities/es/index.js" for import "createOverflowHandler"]

### Steps to reproduce

1. Enable VoiceOver on macOS (Cmd + F5)
2. Render a <Search /> component in a Carbon React app
3. Use Tab to focus into the <Search /> field
4. Attempt to type letters (e.g., a, b, c)
5. Observe that: Space and Backspace trigger events, Character keys are ignored

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by maradwan26** [comment]
Hey, @jorienoll! Seems like this issue occurred in carbon v1.78.2. @heloiselui, @2nikhiltom and I couldn't reproduce it in storybook. Can you confirm if this still exists in the latest version and/or provide more context (perhaps a video)?

**Comment by heloiselui** [comment]
Hey, could you confirm if this is still exists in the latest version? Thank you!

**Comment by heloiselui** [comment]
Thanks again for opening this issue. We're closing it for now as we couldn't reproduce the problem. If it happens again or you find more information, please feel free to reopen it.
